src/psee_evaluator.py

```diff
@@ -30,8 +35,9 @@ def main():
     parser = argparse.ArgumentParser(prog='psee_evaluator.py')
     parser.add_argument('gt_folder', type=str, help='GT folder containing .npy files')
     parser.add_argument('dt_folder', type=str, help='RESULT folder containing .npy files')
+    parser.add_argument('--camera', type=str, default='MOOREA', help='ATIS (QVGA) or MOOREA (720p)')
```

Comment:
not sure we can use names as MOOREA and ATIS. (I am quite sure ATIS is wrong, it refers to pixels with EM+CD, independently of resolution) 
to be consitent with paper we should say Gen1 and Gen4

Reply:
done